CMakeLists.txt

```diff
@@ -53,7 +53,6 @@ zephyr_include_directories(
   ${SOC_DIR}/${ARCH}/${SOC_PATH}
   ${SOC_DIR}/${ARCH}/${SOC_PATH}/include
   ${SOC_DIR}/${ARCH}/${SOC_FAMILY}/include
-  ${BOARD_DIR}
```

Comment:
The commit message should explain why this is removed.

(And probably, this should be in it's own commit)

Reply:
oops, that snuck in, wasn't meant to be part of this PR.